python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -794,7 +796,7 @@ def reset_index(
             self.index = cudf_series.index
             return None
 
-    def contains_properly(self, other, align=True):
+    def contains_properly(self, other, align=True, allpairs=False):
```

Comment:
Looking at geopandas APIs, I see a lot of named mode parameters rather than boolean. how is commonly used, e.g.

lanes_districts = gpd.sjoin(districts, bike_lane, how='inner', predicate='intersects')
lanes_districts

So should we use how="allpairs" (and how="pairwise") instead of a Boolean parameter? Perhaps how might be needed also for other things, so maybe we need a new name for this, e.g. parallelism="allpairs" with options of "all_pairs", "pairwise", "one_to_all", etc.

Thoughts?

Reply:
I will rename from `mode` to `how`.

Reply:
I've actually renamed the parameter back to `allpairs=True` for now. I will file an issue to rename it to `how=` but I'd like to leave it as the boolean because that is the API for the GTC talk. Let's give the GTC talk then give it a week or two before renaming the API.